src/templates/python

```diff
@@ -2,6 +2,8 @@
 ":" /* << "EOF"
 This file is a bash/node polyglot. This is needed for a few reasons:
 
+TODO: We don't support node < 18 anymore, so maybe we can remove this?
```

Comment:
I think that we'll want to use this same mechanism for the `--experimental-wasm-stack-switching` flag soon, so I think it's fine to leave it.